### PR TITLE
Configurations for Axis, Tooltips and chart configurations

### PR DESCRIPTION
**Pull request by ghost**
- Adding configurations options to ng2c3 for Axis,tooltips and chart
- Only Data attribute is mandatory, hence all other input validations fields which are optional are done after checking data attribute

/cc @arunkumars08 @mageshwaranr

### CODE REVIEW

File: app/app.component.ts [hunk 1]
```diff
@@ -9,20 +9,61 @@ import {Ng2C3} from './library/ng2c3.component';
 @Component({
     selector: 'my-app',
     template: `<div style='width:500px;height:500px'>
-                    <ng2-c3 [data]='_data'></ng2-c3>
-                </div>`
+                    <ng2-c3 [data]='_data' [axis]='_axis' [tooltip]='_tooltip' [chartConfig]="_chartConfig"></ng2-c3>
```
Comment (arunkumars08) [inline app/app.component.ts]: Nothing big in difference but it gives consistency if you use same style quotes across.

File: app/app.component.ts [hunk 2]
```diff
@@ -9,20 +9,61 @@ import {Ng2C3} from './library/ng2c3.component';
...
+        },
        } 
-   }
+       
+       //Specific Axis Configuration
+       this._axis = {
+            y2: {
+                show: true
+            }
+        }
```
Comment (arunkumars08) [inline app/app.component.ts]: Although it doesn't seem to have much of an impact, to follow proper practices, it is always good to end things with ';'

File: app/app.component.ts [hunk 3]
```diff
@@ -9,20 +9,61 @@ import {Ng2C3} from './library/ng2c3.component';
...
+        
+        //Specific tooltip Configuration
+        this._tooltip = {
+            format: {
+                title: function (d) { return 'Data ' + d; }
+            }
+        }
```
Comment (arunkumars08) [inline app/app.component.ts]: Although it doesn't seem to have much of an impact, to follow proper practices, it is always good to end things with ';'.
Here and in all the required places.

**Review by arunkumars08 — COMMENTED** [review]
I have given a few minor picks, would be great if they are addressed.

**Comment by ghost** [comment]
@arunkumars08 : Addressed all. Please re-review.

**Comment by arunkumars08** [comment]
LGTM 👍

**Pull request merged**